openhcl/virt_mshv_vtl/src/processor/mshv/arm64.rs

```diff
@@ -356,7 +361,9 @@ impl AccessCpuState for UhProcessor<'_, HypervisorBackedArm64> {
         if self.backing.cpu_state.sp.is_some() {
             expensive_regs.push((HvArm64RegisterName::XSp, self.sp()));
         }
-        self.runner.set_vp_registers(expensive_regs).unwrap();
+        self.runner
+            .set_vp_registers(expensive_regs, GuestVtl::Vtl0)
+            .unwrap();
```

Comment:
How is this AccessCpuState trait used? Any concerns about hardcoding to VTL 0?

Reply:
Seems like this is used on the emulation path. We should follow up (could be a separate change)

Reply:
As you said below: used in emulation, and I opened a bug to make it VTL-aware.